accel-pppd/auth/auth_chap_md5.c

```diff
@@ -81,6 +81,8 @@ static void chap_recv(struct ppp_handler_t *h);
 static void chap_timeout_timer(struct triton_timer_t *t);
 static void chap_restart_timer(struct triton_timer_t *t);
 
+static pthread_rwlock_t config_modify = PTHREAD_RWLOCK_INITIALIZER;
```

Comment:
config loading is out of module scope, instead of boilerplate locking in every module, better to enforce reload event serialization

Reply:
This lock will avoid situation which lead to concurrent running of load_config. Conf_lock is reader function and will not block other reader (second copy of load_config). So we need local write lock in module instead of locking whole config reload process between modules.
Also, this lock can be used to lock local code, sensitive for config reload (such as list_xxx calls).